src/main/java/com/iopipe/IOpipeService.java

```diff
@@ -263,13 +263,20 @@ final RemoteResult __sendRequest(RemoteRequest __r)
 			
 			RemoteResult result = this.connection.send(__r);
 			
-			// Report what was received
-			_LOGGER.debug(() -> "Recv (" + result.code() + "): " +
-				result.body());
-			
 			// Only the 200 range is valid for okay responses
 			if ((result.code() / 100) != 2)
```

Comment:
Question: if this is meant to be "only 200 range" why not have it as `199 < result.code() < 300` as the check case?

Reply:
I used integer division by 100 and a comparison to 2 to make it shorter with just a single comparison rather than expanding it to two.

You can do alternatively:

```
int code = result.code();
if (code >= 200 && code < 300)
```